app/src/parser.c

```diff
@@ -489,22 +489,61 @@ parser_error_t parser_getItem(parser_context_t *ctx,
             CHECK_PARSER_ERR(index_headerpart(parser_tx_obj.header, header_ttl, &ctx->offset));
             uint64_t value = 0;
             CHECK_PARSER_ERR(readU64(ctx,&value));
-            value /= 60000;
-            char tmpBuffer[100];
-            if(value >= 60){
-                uint64_t hours = value/60;
-                if(hours >= 24){
-                    uint64_t days = hours/24;
-                    fpuint64_to_str(tmpBuffer, sizeof(tmpBuffer), days, 0);
-                    snprintf(outVal, outValLen, "%sday", tmpBuffer);
-                    return parser_ok;
+            value /= 1000;
+            char buffer[100];
+            MEMZERO(buffer,sizeof(buffer));
+            uint16_t index = 0;
+            uint64_t days = value / (60*60*24);
+            if(days > 28){
```

Comment:
Is the limit 29 or 28?
Example, is 28days 23 hours valid?

Reply:
checked with them and what we have is fine for now